Signal/src/ViewControllers/NotificationSettingsViewController.m

```diff
@@ -53,6 +53,10 @@ - (void)updateTableContents
                                    OWSSoundSettingsViewController *vc = [OWSSoundSettingsViewController new];
                                    [weakSelf.navigationController pushViewController:vc animated:YES];
                                }]];
+    [soundsSection addItem:[OWSTableItem switchItemWithText:NSLocalizedString(@"NOTIFICATIONS_SECTION_INAPP", nil)
```

Comment:
Since you're changing the copy of `NOTIFICATIONS_SECTION_INAPP`, should we add a comment now too?

Reply:
Good call - done!